logic/src/androidMain/kotlin/com/wire/kalium/logic/util/PlatformDateTimeUtil.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.kalium.logic.util
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import kotlinx.datetime.Instant
+import kotlinx.datetime.toJavaInstant
+import java.text.SimpleDateFormat
+import java.time.format.DateTimeFormatter
+import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
+
+actual open class PlatformDateTimeUtil actual constructor() {
```

Comment:
maybe PlatformDateTimeUtil can be moved to the Util module just in case it is needed somewhere outside logic

Reply:
That's why I'm trying to do right now and it's still WIP, but thanks for writing that, now I'm more sure that it should be this way 😄